int32 tagID
float32 lspeed
float32 rspeed